Analyse_Birds.qmd

```diff
@@ -12,57 +12,419 @@ output:
 
-Taux de succès 
-Taux de survie
+These observations are the following:
 
-voir si habitat préférentiel, région,…
-voir quelles facteurs expliquent taux de succès
+- ***State***: Illinois or Wisconsin
+- ***County***: Dane (Wisconsin), Grant (Wisconsin), Green (Wisconsin), Iowa (Wisconsin), Lafayette (Wisconsin), Monroe (Wisconsin), Rock (Wisconsin), Lee (Illinois), Ogle (Illinois), Will (Illinois), Grundy (Illinois), Carroll (Illinois), and Jo Daviess (Illinois)
+- ***Habitat***: Cool-season Grassland, Pasture, Prairie, Warm-season Grassland
+- ***Species***: Bobolink, Dickcissel, Eastern Meadowlark, Grasshopper Sparrow, Henslow’s Sparrow, Savannah Sparrow, Vesper Sparrow, and Western Meadowlark
+- ***Years***: from 1991 to 2011
+- ***Nest Fate***
+- ***Initiation Date***
+- ***Number fledged***
+- ***Nest End Date***
 
-voir si décalages entre dates de départ du nid -> facteurs expliquant 
 
-voir si habitat préferentiel 
 
-MAIS ATTENTION à ne pas être biaisé : du genre que des oiseaux d’une certaines espèces ne soit présents que dans un habitat
+All theses observations are presented in a table with the following collums : 
 
-refaire analyse de bases cf ASA 
+- *Nest_ID* = unique nest identifier
+- *State* = State in which nest was located (Illinois or Wisconsin)
+- *County* = County in the state in which nest was located
+- *Habitat* = Grassy habitat in which nest was located (pasture, cool-season grassland, warm-season grassland, or prairie)
+- *Species* = English common name
+- *Year* = Year nest was found
+- *Nest_Fate* = Successly fledged at least one young (1) or Failed (0).
+- *Initiation_Date* = Coded date when the first egg was laid. April 1 = 1; May 1 = 31; June 1 = 62; July 1 = 92; August 1 = 123; September 1 = 154
+- *Number_Fledged* = For a successful nest, number of young in nest on nest end date; For a failed nest, Number_Fledged is 0
+- *Nest_End_Date* = Coded date when the nest failed or was successful. April 1 = 1; May 1 = 31; June 1 = 62; July 1 = 92; August 1 = 123; September 1 = 154
 
-# Document R modifié et avec ajouts
+# What kind of questions can we rise ?
 
-## Import des données
 
-**Do not forget to set your working directory in the right place**
-**Menu : Session/Set Working Directory/Choose directory**
-*And indicate Git/2024_MODE_OCR*
+```{r,include=FALSE}
+# Datatset import under the name of "data"
+data <- read.csv("Bird_dataset.csv", sep=";", header = TRUE)
 
+# Selecting the data of interest (here our study is going to focus on the state of illinois)
```

Comment:
Maybe add a comment before stating that you are going to focus on the state of Illinois and explain why you do that.

Reply:
i added that before the whole analysis